src/jvm/main/org/jetbrains/kotlinx/lincheck/trace/TracePoint.kt

```diff
@@ -245,7 +245,16 @@ internal class MethodCallTracePoint(
     val callType: CallType = CallType.NORMAL,
     val isSuspend: Boolean
 ) : CodeLocationTracePoint(eventId, iThread, actorId, codeLocation) {
-    var returnedValue: ReturnedValueResult = ReturnedValueResult.NoValue
+    
+    private var myReturnedValue: ReturnedValueResult? = null
```

Comment:
The naming convention in Kotlin in this case would be `_returnedValue` and `returnedValue` for private/public properties respectively.

Reply:
This is removed after last refactor